waveformCompare.py

```diff
@@ -2128,16 +2128,19 @@ def generate_tags(event):
     orig = event.preferred_origin() or event.origins[0]
     time_tag = orig['time'].isoformat()[:19].replace(':','')+'Z'
 
+    #. Organize in Year and Monthly folders, i.e. '2017/09/GCMT_2017-09-23T125302_6.05_OAXACA_MEXICO'
+    output_path_Alt=output_path+"/"+(orig['time'].isoformat()[:4])+"/"+(orig['time'].isoformat()[5:7])

```

Comment:
For all future work in Python please get used to sticking to PEP8 as soon as possible. Information about this can be found here: http://docs.obspy.org/coding_style.html#obspy-coding-style-guide

Especially valuable are the git pre-commit flake8 hooks that make sure your commits adhere to PEP8.

CC @ngrayluna

Reply:
I'd change this line to..

```python
output_path_alt = '/'.join(output_path, str(orig['time'].year),
                           str(orig['time'].month))
```
..or..

```python
output_path_alt = '{}/{}/{}'.format(output_path, str(orig['time'].year),
                                    str(orig['time'].month))
```